ArduPlane/Log.cpp

```diff
@@ -76,7 +77,8 @@ void Plane::Log_Write_Performance()
         g_dt_max        : perf.G_Dt_max,
         g_dt_min        : perf.G_Dt_min,
         log_dropped     : DataFlash.num_dropped() - perf.last_log_dropped,
-        hal.util->available_memory()
```

Comment:
It's funny that this line doesn't have "mem_avail : " and "load :" at the front to specify the field.  I guess it works but it doesn't quite match the style we normally use.  Not saying this should stop the PR from going in but personally I'd fix that.